### PR TITLE
add some libkonq handling inside dolphin and ark

### PR DESCRIPTION
**Pull request by karolherbst**

### CODE REVIEW

File: kde-apps/ark/ark-5.9999.ebuild [hunk 1]
```diff
@@ -32,6 +32,7 @@ DEPEND="
 	$(add_frameworks_dep kservice)
 	$(add_frameworks_dep kwidgetsaddons)
 	$(add_frameworks_dep kxmlgui)
+	$(add_kdeapps_dep libkonq)
```
Comment (kensington) [inline kde-apps/ark/ark-5.9999.ebuild]: Should be at the bottom and conditional on dolphin.

File: kde-apps/ark/ark-5.9999.ebuild [hunk 2]
```diff
@@ -39,6 +40,13 @@ DEPEND="
 "
 RDEPEND="${DEPEND}"
 
+src_prepare() {
```
Comment (kensington) [inline kde-apps/ark/ark-5.9999.ebuild]: Should be src_configure.

File: kde-apps/ark/ark-5.9999.ebuild [hunk 3]
```diff
@@ -39,6 +40,13 @@ DEPEND="
...
+	mycmakeargs=(
```
Comment (kensington) [inline kde-apps/ark/ark-5.9999.ebuild]: Please declare local.

**Comment by karolherbst** [comment]
@kensington  done

**Comment by kensington** [comment]
Thanks!

**Pull request merged**